Client/game_sa/CModelInfoSA.cpp

```diff
@@ -516,7 +516,7 @@ bool CModelInfoSA::IsValid()
 {
     if (m_dwModelID >= 20000 && m_dwModelID < MODELINFO_MAX)
         return true;
-    return ppModelInfo[m_dwModelID] != 0;
+    return ppModelInfo[m_dwModelID] != 0 && pGame->GetStreaming()->GetStreamingInfoFromModelId(m_dwModelID)->sizeInBlocks;
```

Comment:
could you split this up onto separate lines? something like this
```suggestion
    if (ppModelInfo[m_dwModelID] == nullptr)
        return false;

    auto sizeInBlocks = pGame->GetStreaming()->GetStreamingInfoFromModelId(m_dwModelID)->sizeInBlocks;
    return sizeInBlocks > 0;
```